app/viewmodels/CashAccountV2ViewModel.scala

```diff
@@ -160,4 +169,20 @@ object CashAccountV2ViewModel {
         postLinkMessage = Some("cf.cash-account.help-and-support.link.text.post")))
     )
   }
+
+  private def populatePaginationModel(pageNo: Option[Int],
+                                      totalDailyStatementsSize: Int)
+                                     (implicit config: AppConfig) = {
+    val shouldPaginationBeDisabledAndShowAllRecords = totalDailyStatementsSize <= config.numberOfRecordsPerPage
```

Comment:
this is very long, maybe keep it short like `isPaginationDisabled`

Reply:
Done